projects/m1/006-sum-of-digits-in-a-integer/solutions/andreaValle/006-sum-of-a-digits-in-a-integer.py

```diff
@@ -0,0 +1,17 @@
+#this function will do all the work for us
+def calculation():
```

Comment:
You are already using functions, but it's too early given the amount of exercise you have done.

If you feel comfortable with that go ahead, otherwise try to stay consistent with the knowledge suggested in the exercise